src/components/CategoryVideos/CategoryVideos.styles.tsx

```diff
@@ -0,0 +1,32 @@
+import styled from '@emotion/styled'
```

Comment:
Let's not include _anything_ related to video categories in this PR, this file included. If we want a place to test out the filters bar let's just create a playground with only the necessary logic there

Reply:
hmm I will remove manually the files then and later try a rebase without it